txnkv/transaction/2pc.go

```diff
@@ -567,6 +571,20 @@ func (c *twoPhaseCommitter) doActionOnMutations(bo *retry.Backoffer, action twoP
 	// check the number of batches. However we don't want the check fail after any code changes.
 	c.checkOnePCFallBack(action, len(groups))
 
+	// set delete only operations allowed when tikv disk almost full.
```

Comment:
I prefer to remove this logic because it's expensive and depends on something undefined. Let's find a better way to tell delete-only transactions in TiDB.

Reply:
Can we use some config online set to control this logic?

Reply:
we can conclude if it is delete only in `initKeysAndMutations`

Reply:
Good Place

Reply:
good place